README.md

```diff
@@ -31,13 +31,23 @@ The tool can be installed as a standalone command line tool. The following depen
 |Python|3.10|
 |VCFtools|0.1.16|
 
-### Install directly from GitHub
+Additionally, the `libmagic` C library must also be installed on the system.
```

Comment:
Consider adding some notes on how to install `libmagic` on different systems. For example, the CPAN module author provides [this](https://github.com/gitpan/File-LibMagic/blob/master/INSTALL.md).

Reply:
Makes sense, I added some instructions on installing `libmagic`